spec/inputsOutputs/inputsOutputs.spec.js

```diff
@@ -136,20 +143,18 @@ describe('inputsOutputs page', function() {
       expect(components.layout.isElementDisplayed(element(by.css('.popover')))).toBe(true);//popover should show up
 
       components.popovers.popover.clickNo();//click no btn
-      expect(components.layout.isElementDisplayed(element(by.css('.popover')))).toBe(false);//popover should hide
```

Comment:
we should test clicking no, replace "isElementDisplayed" with protractor isPresent method.